incubator/hnc/Makefile

```diff
@@ -182,18 +182,22 @@ kind-deploy:
 	CONFIG=kind $(MAKE) deploy
 
 ###################### E2E TEST #########################
+# This test will run on the current cluster the user deployed (either kind or 
+# a kubernetes cluster). Please make sure to have the test cluster ready 
+# before running the test.
+
 # Sleep time is added for the webhook to get ready before test. The time 
 # comes from watching the log as well as multiple experiments. 
 .PHONY: e2e-test
-e2e-test: deploy
+e2e-test: 
 	go clean -testcache
 	sleep 15
```

Comment:
Do we need the "sleep" now that we're asking the user to set up the cluster ahead of time?

Reply:
Ah right. And agreed that we don't need 2 targets now.

Reply:
Done